scripts/skills/effects/rf_encumbrance_effect.nut

```diff
@@ -125,51 +84,53 @@ this.rf_encumbrance_effect <- ::inherit("scripts/skills/skill", {
 		return 0;
 	}
 
-	function onUpdate( _properties )
+	function getMovementFatigueCostModifier( _encumbranceLevel )
 	{
-		switch (this.getEncumbranceLevel())
+		switch (_encumbranceLevel)
 		{
-			case 0:
-			case 1:
-				return;
+			case 0:	// Not a valid level
```

Comment:
Minor comment: 
If by `not valid` you mean that no penalties are applied at this level then in my opinion it would be better to say `Not applicable at this level`. Because `not valid` seems to imply that it's wrong or unacceptable. Or you could say `Not a valid level for this effect` to be clear that it's only for this effect that it's not valid.